Dockerfile

```diff
@@ -7,17 +7,23 @@ FROM debian:${DEBIAN_VERSION}-slim AS base
 ARG RUNDIR
 ARG ENTRYPOINT=/bin/bash
 ARG RUNTIME_PACKAGES
+ARG RUNTIME_TAR_PACKAGES
 
 RUN apt update -y && \
     apt install -y --no-install-recommends \
         libreadline8 \
         busybox \
         netcat-openbsd \
         procserv \
+        wget \
         $RUNTIME_PACKAGES && \
     apt clean && \
     rm -rf /var/lib/apt/lists/*
 
+COPY --from=build-image /usr/local/bin/lnls-get-n-unpack /usr/local/bin/lnls-get-n-unpack
+RUN lnls-get-n-unpack -r $RUNTIME_TAR_PACKAGES && \
+    ldconfig
+
```

Comment:
This only makes sense if you are downloading the same/similar tarball while building base, or if RUNTIME_TAR_PACKAGES are installed at build time as well when you're building the final IOC... Do you agree? (mostly just wondering if for commit internal consistency, we should add something else here)

Reply:
Not necessarily, although I doubt it is the case for us now. A runtime script might depend on other utility (not packaged for Debian), which isn't required during the build (a vendor executable, for instance). Being possible to call system applications in IOC Shells suggests that this situation can occur.

What do you propose regarding this fact of probably always installing packages also used in other build phases?

Reply:
That the build-stage preparation for build should maybe include a similar RUN (without the ldconfig, though?)

I see your point and could leave this commit as is, too.

Reply:
Well, you have a good point. When a library is not built in the base image, it will require a way to install during the `build-stage` as well. I've added it now. Thanks for the suggestion.